app/user-data-download/server/cronProcessDownloads.js

```diff
@@ -328,22 +342,28 @@ const makeZipFile = function(exportOperation) {
 		return;
 	}
 
-	const output = fs.createWriteStream(targetFile);
+	const zipFilePromise = new Promise((resolve, reject) => {
```

Comment:
instead of creating a Promise here and `await` for it, you can just return it, so the callee will `await` for it (and it already has the `await`)